images/arm-img-builder.sh

```diff
@@ -183,6 +188,18 @@ while [ "$#" -gt 0 ]; do
     shift 1
 done
 
+if [ -z "$cos_config"]; then
+  echo "No cos-config file specified"
```

Comment:
Maybe sticking to default instead of existing here? I guess it's fine to keep the current labels as default ( so no need to pass by a default config each time)

Reply:
I coded it to be consistent with the other options and because I am not sure in which context this script it used, so I am not sure what would be a sane default cos-config file value ( it could be `packages/cos-config/cos-config`) but I feels odd to me here using some relative path making the script dependent on the working dir. For the x86 script this is solved at Makefile level. The makefile manages the default.

Reply:
Sorry my bad I wasn't clear! I mean defaults for `${ACTIVE_LABEL}` , `${PASSIVE_LABEL}`, etc so the config file is not mandatory, but rather optional overriding defaults values (`like COS_ACTIVE` for `ACTIVE_LABEL` and so on)

Reply:
I am not really a fan of this idea, to be honest, since this means going back to hard coded values and having the same default written in more than one place, but I could live with it as long as our CI does not makes use of them and we enforce passing the config file, which is already the case for the x86 builder. Gonna add a commit for it.

Anyway I hope we can get rid of all this once we have defaults defined within the elemental client and some structured configuration files to alter them. Then these scripts should only make use of the elemental client.

Reply:
yep, I do agree it's not ideal now that we have all the defaults fragmented, but with the new cli this is going to be eased out quite a lot as we will have a central place (so no dups).

The idea behind the arm script is that we can re-use them as a base for the user-facing ones in the future, and ideally the minimum input from the user would be an image (at the moment it's hardcoded to the odroid, but should probably just point to the generic image instead) to get started